package.json

```diff
@@ -23,6 +23,7 @@
     "bootstrap": "npm i && lerna bootstrap",
     "release": "npm run build:full && lerna publish",
     "update-template-deps": "node bin/update-template-deps -f",
+    "version": "npm run update-template-deps",
```

Comment:
What triggers the `version` script?

Reply:
`lerna publish` will trigger the `version` scripts as follows:

1. Run `preversion` at root
2. Run `preversion` for each package
3. Update package.json and CHANGELOG for each package
4. Run 'version` for each package
5. Run `version` at root

Reply:
It follows https://docs.npmjs.com/cli/version